shared/event_logging_tracker.js

```diff
@@ -1,82 +1,53 @@
 const VIEWPORT_TRACKING_IDENTIFIER = 'viewport_tracking';
 const VIEWPORT_TRACKING_CLOSED_EVENT_IDENTIFIER = 'vtc';
 
-export default class EventLoggingTracker {
+export const EVENT_SCHEMA = 'event.WMDEBannerEvents';
+export const BANNER_SIZE_SCHEMA = 'event.WMDEBannerSizeIssue';
 
-	constructor( bannerName ) {
-		this.bannerName = bannerName;
-	}
+const ALWAYS_TRACK = 1;
```

Comment:
```suggestion
export const ALWAYS_TRACK = 1;
```

This can then also be used in banner code, making it more readable